docs/v2/spec/2018_Modified_LAR_Code_Sheet.csv

```diff
@@ -0,0 +1,246 @@
+﻿Data Field Number,Field Name,Codes,Descriptions,Notes
+3,Loan Type,1,"Conventional (not insured or guaranteed by FHA, VA, RHS, or FSA)",
+3,Loan Type,2,Federal Housing Administration insured (FHA),
+3,Loan Type,3,Veterans Affairs guaranteed (VA),
+3,Loan Type,4,USDA Rural Housing Service or Farm Service Agency guaranteed (RHS or FSA),
+4,Loan Purpose,1,Home purchase,
+4,Loan Purpose,2,Home improvement,
+4,Loan Purpose,31,Refinancing,
+4,Loan Purpose,32,Cash-out refinancing,
+4,Loan Purpose,4,Other purpose,
+4,Loan Purpose,5,Not applicable,
+5,Preapproval,1,Preapproval requested ,
+5,Preapproval,2,Preapproval not requested,
+6,Construction Method,1,Site-built,
+6,Construction Method,2,Manufactured Home,
+7,Occupancy Type,1,Principal residence,
+7,Occupancy Type,2,Second residence,
+7,Occupancy Type,3,Investment property,
+9,Action Taken,1,Loan originated,
+9,Action Taken,2,Application approved but not accepted,
+9,Action Taken,3,Application denied,
+9,Action Taken,4,Application withdrawn by applicant,
+9,Action Taken,5,File closed for incompleteness,
+9,Action Taken,6,Purchased loan,
+9,Action Taken,7,Preapproval request denied,
+9,Action Taken,8,Preapproval request approved but not accepted,
+13,Ethnicity of Applicant or Borrower,1,Hispanic or Latino,
+13,Ethnicity of Applicant or Borrower,11,Mexican,
+13,Ethnicity of Applicant or Borrower,12,Puerto Rican,
+13,Ethnicity of Applicant or Borrower,13,Cuban,
+13,Ethnicity of Applicant or Borrower,14,Other Hispanic or Latino,
+13,Ethnicity of Applicant or Borrower,2,Not Hispanic or Latino,
+13,Ethnicity of Applicant or Borrower,3,"Information not provided by applicant in mail, internet, or telephone application.",Code 3 only valid for ethnicity of Applicant or Borrower: 1.
+13,Ethnicity of Applicant or Borrower,4,Not applicable.,Code 4 only valid for Ethnicity of Applicant or Borrower: 1.
+18,Ethnicity of Co-Applicant or Co-Borrower,1,Hispanic or Latino,
+18,Ethnicity of Co-Applicant or Co-Borrower,11,Mexican,
+18,Ethnicity of Co-Applicant or Co-Borrower,12,Puerto Rican,
+18,Ethnicity of Co-Applicant or Co-Borrower,13,Cuban,
+18,Ethnicity of Co-Applicant or Co-Borrower,14,Other Hispanic or Latino,
+18,Ethnicity of Co-Applicant or Co-Borrower,2,Not Hispanic or Latino,
+18,Ethnicity of Co-Applicant or Co-Borrower,3,"Information not provided by applicant in mail, internet, or telephone application. ",Note: Code 3 only valid for ethnicity of Co-Applicant or Co-Borrower: 1.
+18,Ethnicity of Co-Applicant or Co-Borrower,4,Not applicable. ,Note: Code 4 only valid for Ethnicity of Co-Applicant or Co-Borrower: 1.

```

Comment:
Remove period after `Not applicable`.

Remove periods after anything else in this column that is not a sentence.

Reply:
Periods have been removed in general. Not necessary for a code sheet.